R/dist.R

```diff
@@ -197,16 +198,16 @@ dist_skel <- function(n, dist = FALSE, cum = TRUE, model,
 #'   cores = ifelse(interactive(), 4, 1), verbose = TRUE
 #' )
 #' }
-dist_fit <- function(values = NULL, samples = NULL, cores = 1,
+dist_fit <- function(values = NULL, samples = 1000, cores = 1,
                      chains = 2, dist = "exp", verbose = FALSE) {
-  if (is.null(samples)) {
-    samples <- 1000
-  }
-
   if (samples < 1000) {
     samples <- 1000
   }
 
+  warning(sprintf("%s %s", "`samples` must be at least 1000.",
```

Comment:
This needs to be inside 203 to 205 to it will always trigger.

Reply:
That's true. Will change asap.